src/deepsparse/v2/routers/router.py

```diff
@@ -88,8 +95,13 @@ def validate(operators: List[Operator]) -> Union[bool, str]:
                 continue
 
             if current_output_schema != next_input_schema:
+                # TODO: Log error message below
+                """
                 return (
                     f"Operator at idx {idx}: {type(operators[idx])} has invalid "
                     f"output schema {current_output_schema} for next operator "
                     f"{type(operators[idx + 1])} which requires {next_input_schema}"
                 )
+                """
+                return False
+        return True
```

Comment:
Why was this commented out?

Reply:
this returns a boolean, not a string. The strings should be logs, once include a logger.